compiler/compiler.go

```diff
@@ -700,6 +700,15 @@ func compile(n semantic.Node, subst semantic.Substitutor) (Evaluator, error) {
 		if err != nil {
 			return nil, err
 		}
+		// Rewrite of "fake function" to vectorize constants.
+		if c, ok := n.Callee.(*semantic.IdentifierExpression); ok && c.Name.LocalName == "~~vecRepeat~~" {
```

Comment:
named `~~vecRepeat~~` so as to be illegal as a symbol coming from userland (ie the flux source itself), thus making it impossible to clash. The choice to use double-enclosing tilde was just a nod to the fact the function call was being erased and replaced: ~~vecRepeat~~.